packages/editor-ui/src/components/mixins/pinData.ts

```diff
@@ -32,12 +32,16 @@ export const pinData = (Vue as Vue.VueConstructor<Vue & PinDataContext>).extend(
 				const positionMatch = error.message.match(/at position (\d+)/);
 
 				error.message = message.charAt(0).toUpperCase() + message.slice(1);
+				error.message = error.message.replace(
+					'Unexpected token \' in JSON',
+					'Unexpected single quote. Please use double quotes (") instead',
```

Comment:
i18n?

Reply:
`JSON.parse` messages are english only afaik. Only way to do i18n is to detect what error it is, parse its parts, and assign a translation key to it.

Reply:
I've added partial translation. The standard error message will still be in English, but this specific case will be translated. I suppose it makes sense, since it will occur more often than the others.